src/lua/zencode_w3c.lua

```diff
@@ -297,3 +333,28 @@ When(
                                           encoding = 'complex' })
     end
 )
+
+When(
+    "create the '' from ''",
```

Comment:
should be less generic:
```
create the public key '' from did document ''
```

Reply:
Yes, you are right, it should be less generic. Two doubts:
- This statement creates an object called `public key` right? 
- Moreover, since it creates only public keys, this statement will not be used to extract  the `ethereum adrress`?